main.py

```diff
@@ -1,8 +1,64 @@
-from fastapi import FastAPI
+from fastapi import Body, FastAPI
 
 app = FastAPI()
 
+BOOKS = [
+    {"title": "Physics", "author": "Some random dude", "category": "science"},
+    {"title": "Computer", "author": "Another random dude", "category": "Science"},
+    {"title": "English", "author": "Some british guy", "category": "Literature"},
+]
 
-@app.get("/")
-async def hello_world():
-    return {"message": "Hello_world"}
+
+@app.get("/books")
+async def get_books() -> list[dict[str, str]]:
+    return BOOKS
```

Comment:
Docstring are missing in all functions.

Reply:
Ahsan Bhai I asked in the meet and you said we can skip doc string for routes.

Reply:
My bad